sw/device/silicon_creator/lib/ownership/isfb.c

```diff
@@ -34,7 +34,7 @@ rom_error_t isfb_boot_request_process(const manifest_ext_isfb_t *ext,
     return kErrorOwnershipISFBNotPresent;
   }
   const owner_isfb_config_t *isfb = owner_config->isfb;
-  if (ext->product_expr_count * 2 > isfb->product_words ||
+  if (ext->product_expr_count > isfb->product_words ||
```

Comment:
@moidx I didn't understand why the code said `ext->product_expr_count * 2 > isfb->product_words`.